tools/SeekRPC/seek_rpc.thrift

```diff
@@ -27,15 +41,24 @@ struct StringDoublePair {
     2: required double value;
 }
 
+enum QueryStatus {
+    Complete = 1,
+    Incomplete,
+    Error,
+}
+
 struct QueryResult {
     1: required bool success;
     2: required list<StringDoublePair> gene_scores;
     3: optional list<StringDoublePair> dataset_weights;
-    4: optional string statusMsg;
+    4: optional QueryStatus status;
+    5: optional string statusMsg;
```

Comment:
Variable `statusMsg` doesn't follow other variables' name convention in the same struct.

Reply:
Yes good point. In looking at other pre-existing cpp files (like SeekCentral.h) it seems mostly camel case is used but occasionally snake case. I'm thinking to change everything in this RPC definition to camel case. I think it's early enough that it probably won't be too disruptive. What do you think?

Reply:
Agree. If changing it's definitely the sooner the easier.